OWNERS_ALIASES

```diff
@@ -204,6 +204,17 @@ aliases:
   provider-vmware:
     - cantbewong
     - frapposelli
+  publishing-bot-approvers:
+    - dims
+    - nikhita
+    - sttts
+  publishing-bot-reviewers:
+    - cpanato
+    - jeremyrickard
+    - justaugustus
+    - puerco
+    - saschagrunert
+    - xmudrii
```

Comment:
not missing veronica?

Reply:
> not missing veronica?

@cpanato -- This was intentional :)
Her [onboarding issue](https://github.com/kubernetes/sig-release/issues/1713) still needs to be completed and I didn't want to draw away from the pub-bot review discussion by adding her here until that issue is wrapped up.

Reply:
got it, thanks!